lib/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/mapper/remove_delegate_mapper.dart

```diff
@@ -1,16 +1,22 @@
 import 'package:seeds/domain-shared/page_command.dart';
 import 'package:seeds/domain-shared/page_state.dart';
 import 'package:seeds/domain-shared/result_to_state_mapper.dart';
-import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_state.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegates_bloc.dart';
 
 class RemoveDelegateResultMapper extends StateMapper {
-  DelegateState mapResultToState(DelegateState currentState, Result result) {
+  DelegatesState mapResultToState(DelegatesState currentState, Result result) {
     if (result.isError) {
       print('Error transaction hash not retrieved');
       return currentState.copyWith(
-          pageState: PageState.success, pageCommand: ShowErrorMessage('Fail to remove delegate'));
+        pageState: PageState.failure,
```

Comment:
Hi Raul, I dont think this should show a fullPageErrorIndicator.

Reply:
oh i see this screen use a snack bar. I'll fix it

Reply:
done